docs/source/Intro.rst

```diff
@@ -7,9 +7,25 @@
 Introduction
 =====================================
 
-Challenges for accurate weather prediction still exist in numerical models. To facilitate the development and improvement of the NOAA `Unified Forecast System (UFS) <https://ufscommunity.org/>`_ Weather Model (WM), case studies are needed based on these forecast challenges. There are several `known model biases in the UFS WM <https://drive.google.com/file/d/1rdFPbY28d7cRrcShy0uo4Mtqwh3BSzYg/view>`_. These known biases include but are not limited to progressive synoptic patters, degraded tropical cyclone tracks, and incapability to simulate stable boundary layer structures.
+Challenges for accurate weather predictions still exist in numerical models. To facilitate the development and improvement of the NOAA operational model, NOAA's Global Forecast System (GFS) is shared with the broad community through the `Unified Forecast System (UFS) <https://ufscommunity.org/>`_ applications. GFS version 15 (v15) was in operation in June 2019 and the planned date for GFS version 16 in operation is in Feb 2021 (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). The UFS Medium-Range Weather (MRW) Application (App) provides the source code of GFS, including physics schemes for GFS.v15 (CompSet ``GFS.v15p2``) and additional developmental schemes for GFS.v16 (CompSet ``GFS.v16beta``). The most up-to-date source codes for GFS.v16 can be accessed from the `ufs-weather-model <https://github.com/ufs-community/ufs-weather-model>`_ GitHub repository. The known biases and development priorities related to GFS.v15 (`Stan et al. 2019 <https://drive.google.com/file/d/1rdFPbY28d7cRrcShy0uo4Mtqwh3BSzYg/view>`_) and GFS.v16 (recently presented by `Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_) have been identified and listed below: 
```

Comment:
I would move the sentence "GFS version 15 ..." before the sentence starting with "The known biases and development ...". And also: GFS version 15 went into operations in June 2019, the operational implementation fo GFS version 16 is currently planned for February 2021.